content/tutorials/r_inboveg/index.Rmd

```diff
@@ -0,0 +1,332 @@
+---
+title: "Tutorial on how to retrieve data from the INBOVEG database"
+author: "Els De Bie, Hans Van Calster & Jo Loos"
+date: 2020-02-11
+categories: ["databases"]
+tags: ["database", "queries"] 
+output: 
+  md_document:
+    preserve_yaml: true
+---
+
+```{r setup, include=FALSE}
+library(knitr)
+opts_chunk$set(echo = TRUE)
+```
+
+# Introduction
+The Flemish vegetation database, INBOVEG, is an application developed to provide
+a repository of relevés and makes the relevés available for future use.
+
+INBOVEG supports different types of recordings: BioHab recordings (protocol of 
+Natura 2000 monitoring) and the classic relevés. The classic relevés can stand 
+alone, be an element of a collection or element of a chain where the linkage is 
+used to give information about the relative position of recording within a series.
+Ample selection and export functions toward analysis tools are provided. It also
+provides standardized lists of species, habitats, life forms, scales etc. 
+Original observations are preserved and a full history of subsequent 
+identifications is saved.
+
+
+# Aim
+In this tutorial we make functions available to query data directly from the 
+INBOVEG SQL-server database. This to avoid writing your own queries or 
+to copy/paste them from the access-frontend for INBOVEG.
+
+We have provided functions to query
+
+* survey (INBOVEG-projects)
+
+* recordings (vegetation relevés)
+
+* metadata of recordings (header info)
+
+* classification (Natura2000 or local classification like BWK)
+
+* qualifiers (management and site characteristics)
+  
+# Packages and connection
+In order to run the functionalities, some R packags need to be installed. 
+
+The main functions that we will use in this tutorial all start with `inboveg_*`. These functions are made available by loading the `inborutils` package.
+
+
+You can install inborutils from github with:
+
+```{r, eval = FALSE}
+install.packages("devtools")
+devtools::install_github("inbo/inborutils")
+```
+
+This tutorial will only work for people with access to the INBO network. As an INBO employee, you should make sure you have reading-rights for CYDONIA, otherwise place an ICT-call (ict.helpdesk@inbo.be)
```

Comment:
```suggestion
This tutorial will only work for people with access to the INBO network. As an INBO employee, you should make sure you have reading-rights for CYDONIA, otherwise place an ICT-call.
```